src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This file should live in the src/Common/src/Microsoft/Win32/SafeHandles directory, not in the Interop directory